.github/workflows/i18n-weekly-release.yml

```
@@ -0,0 +1,55 @@
+name: I18N Weekly Release
+on:
+  schedule:
+    - cron: '0 20 * * 0'
+
+jobs:
+  i18n-release:
+    name: Release
+    runs-on: ubuntu-20.04
+
+    steps:
+      # clone the repository
+      - uses: actions/checkout@v2
+      # enable dependencies caching (vendor and node_modules are wiped during build so they are ignored here)
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/composer/
+          key: ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm/
+          key: ${{ runner.os }}-npm-${{ hashFiles('package-lock.json') }}
+      # setup PHP, but without debug extensions for reasonable performance
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          tools:       composer
+          coverage:    none
+
+      - name: Build release
+        run: |
+          npm ci
+          npm run build
+
+      - name: Upload release
+        run: |
+          TAG="i18n-$(date +%y%m%d)"
+          NAME="Version for internationalization testing $TAG. Not for production."
+          BODY='This version is intended to test the internationalization strings. It should NOT be used for production.'
+
+          response=$(curl --request POST \
+            --header 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' \
+            --header 'Content-Type: application/json' \
+            --data "{\"tag_name\":\"$TAG\",\"target_commitish\":\"develop\",\"name\":\"$NAME\",\"body\":\"$BODY\",\"prerelease\":true}" \
+            https://api.github.com/repos/Automattic/woocommerce-payments/releases)
+
+          upload_url=$(echo $response | sed 's|.*"upload_url": "\([^{]*\).*|\1|')
```

Somehow this does not work for me. Do I miss something?

```
curl -s --header 'Authorization: token <token>' --header 'Content-Type: application/json' https://api.github.com/repos/Automattic/woocommerce-payments/releases/42980009 | sed 's|.*"upload_url": "\([^{]*\).*|\1|'
```
That gives me a json response somehow. Maybe because the response json is multi-line?

What do you think of utilizing `jq`?
```suggestion
          upload_url=$(echo $response | jq -r '.upload_url' | sed 's/{.*//')
```

---

Yeah, I think this command just needs one `grep 'upload_url'` to find the right line 🤔 

The command works for me locally after I've added that 🙂

---

Yeah, I think this command just needs one `grep 'upload_url'` to find the right line 🤔 

The command works for me locally after I've added that 🙂

---

I don't think relying on `jq` is a good idea since it's an external depenency. I think a combination of `grep` and `sed` would be better here 🤔

---

The response seems to be one line though. `echo $response | sed 's|.*"upload_url": "\([^{]*\).*|\1|')` worked flawlessly for me when I tried to execute this locally with https://github.com/Automattic/woocommerce-payments/releases/tag/i18n-dev-20210521.

We can add the `grep` just to ensure the correct data is there, but I don't think it's necessary.
What do you think?

---

I'm OK with that. Again, not sure why I got multi-line response. It must be something on my end.

---

> I'm OK with that

Do you mean you're ok with adding `grep` or that you're ok with not adding anything? 🙂 

Personally, I think the `grep` could be nice to have — just in case GitHub do start multi-lining their response — but I don't think it's strictly necessary.

---

I'm OK with not using `jq`. Adding `grep` would be nice too, just in case as you said.

---

Added grep command in 04cdcb86bd19fd336ea919c537d26495a9fc3bcb